include/ignition/sensors/ImuSensor.hh

```diff
@@ -136,6 +157,13 @@ namespace ignition
       /// \return Gravity vectory in meters per second squared.
       public: math::Vector3d Gravity() const;
 
+      /// \brief Specify the rotation offset of the coordinates of the World
+      //  frame relative to a geo-referenced frame
```

Comment:
Doxygen looks specifically for 3 slashes `///` when parsing  the API documentation, so please use 3 slashes consistently in header files.

Reply:
Done